### PR TITLE
Use a custom agent that keeps the catalogue API connection alive

### PR DESCRIPTION
**Pull request by alexwlchan**
We're seeing a steady trickle of reset connections between the front-end and the catalogue API; since we know we're always going to be talking to the API, let's try holding the connection open rather than recreating it on every request. Fingers crossed this makes it more reliable!

We do fetch catalogue data asynchronously on the front-end, when we find visually similar images on a work – for that, there's a new endpoint `/api/visuallySimilarImages/[imageId]`. My guess is that this will be beneficial for the user (🤞):

* All connections to the catalogue API come from the server, which can use the keep-alive'd connection
* All connections from the browser go to the front-end app, rather than opening separate connections to the front-end and the catalogue API
* We can toss a bunch of code out of the bundle that's unneeded

### CODE REVIEW

**Comment by alexwlchan** [comment]
Urk.

**Review by davidpmccormick — APPROVED** [review]

**Comment by alexwlchan** [comment]
I want to wait for https://github.com/wellcomecollection/wellcomecollection.org/pull/7765 to merge this, to offset the bundle size gains.

**Comment by alexwlchan** [comment]
I want to see if I can get away without bringing in quite such a bundle size increase before merging 😬

File: catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx [hunk 1]
```diff
@@ -98,6 +98,7 @@ const PhysicalItems: FunctionComponent<Props> = ({
       const updateItemsStatus = async () => {
         const itemsResponse = await fetch(`/api/works/items/${work.id}`, {
           signal,
+          credentials: 'same-origin',
```
Comment (jamieparkinson) [inline catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx:101]: Comment about why this is here / that it's not actually related to credentials?

File: catalogue/webapp/components/VisuallySimilarImagesFromApi/VisuallySimilarImagesFromApi.tsx [hunk 2]
```diff
@@ -36,15 +36,13 @@ const VisuallySimilarImagesFromApi: FunctionComponent<Props> = ({
...
     const fetchVisuallySimilarImages = async () => {
-      const fullImage = await getImage({
-        id: originalId,
-        toggles,
-        include: ['visuallySimilar'],
-      });
-      if (fullImage.type === 'Image') {
+      const fullImage = await getVisuallySimilarImagesClientSide({ id: originalId, toggles });
+
+      if (fullImage && fullImage.type === 'Image') {
```
Comment (jamieparkinson) [inline catalogue/webapp/components/VisuallySimilarImagesFromApi/VisuallySimilarImagesFromApi.tsx:41]:
```suggestion
      if (fullImage?.type === 'Image') {
```

**Review by jamieparkinson — APPROVED** [review]

File: catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx (hunk 1 above)
Comment (alexwlchan) [reply catalogue/webapp/components/PhysicalItems/PhysicalItems.tsx]: Added.

**Pull request merged**